modules/developers-guide/examples/bash/java-connect-gen-auth.sh

```diff
@@ -0,0 +1,3 @@
+curl -X POST localhost:8081/v1/auth/token/generate \
```

Comment:
For consistency shouldn't this be the same as the others?

```sh
curl -L -X POST 'http://localhost:8081/v1/auth' \
  -H 'Content-Type: application/json' \
  --data-raw '{
    "username": "cassandra",
    "password": "cassandra"
}'
```

Reply:
Not used - deleted it.